internals/secrethub/tree.go

```diff
@@ -47,53 +50,90 @@ func (cmd *TreeCommand) Register(r command.Registerer) {
 	clause := r.Command("tree", "List contents of a directory in a tree-like format.")
 	clause.Arg("dir-path", "The path to to show contents for").Required().PlaceHolder(optionalDirPathPlaceHolder).SetValue(&cmd.path)
 
+	clause.Flag("full-paths", "Print the full paths of the directories and secrets.").Short('f').BoolVar(&cmd.fullPaths)
+	clause.Flag("no-indentation", "Print the content without indentation.").Short('i').BoolVar(&cmd.noIndentation)
+	clause.Flag("no-report", "Skip the report at the bottom.").BoolVar(&cmd.noReport)
+	clause.Flag("noreport", "Skip the report at the bottom.").BoolVar(&cmd.noReport)
```

Comment:
While we do want to support both `--noreport` (as exists in the unix tree command) and `--no-report` (more consistent with how we name flags in other commands), we should only show one of them in the help-text. The one we're documenting is the "recommended way". The other we just add so that when you type that because you expect/remember it, it also works.

By excluding the other from the help-text, we keep the help-text concise and easier to understand.

```suggestion
	clause.Flag("no-report", "Skip the report at the bottom.").BoolVar(&cmd.noReport)
	clause.Flag("noreport", "Skip the report at the bottom.").Hidden().BoolVar(&cmd.noReport)
```

Reply:
Nice suggestion. Thanks